### PR TITLE
Parameterize email domain in client

### PR DESCRIPTION
**Pull request by meganniu**
Set email domain in `REACT_APP_EMAIL_DOMAIN` env var

### CODE REVIEW

**Review by kevin-wangg — APPROVED** [review]
lgtm

File: client/src/services/auth.ts [hunk 1]
```diff
@@ -68,6 +68,7 @@ export const createNewAccount = async (
   if (!passwordRequirements.test(password)) {
     errors.password = AuthErrorMessage["invalid-password"];
   }
+  console.log(process.env.REACT_APP_FIREBASE_APP_ID)
```
Comment (chamod-gamage) [inline client/src/services/auth.ts]: stray log

**Review by chamod-gamage — APPROVED** [review]
lgtm - stray log

File: client/src/services/auth.ts (hunk 1 above)
Comment (meganniu) [reply client/src/services/auth.ts]: lol not a oneliner after all

Comment (meganniu) [reply client/src/services/auth.ts]: tyty

**Pull request merged**